### DISCUSSION TITLE
Question about the ciprofloxacin tutorial

### DISCUSSION THREAD

**Discussion by bahsa505** [Open-Systems-Pharmacology/Forum#1195] [Questions & Problems] [answered]
Hi all,

I am working on the ciprofloxacin tutorial

https://github.com/Open-Systems-Pharmacology/ISSX-2017-Cologne/tree/master/Session%202.%20Building%20a%20PBPK%20model

 I keep getting this error message when trying to clone the simulation or send to Mobi, although the simulation itself is working as expected!

<img width="476" alt="Screenshot 2022-04-13 102011" src="https://user-images.githubusercontent.com/61779799/163132539-44d4955c-dc07-41b1-9078-bfebc0a62bce.png">

Any thoughts on what's going wrong there?

Kind regards
Salma

**Comment by tobiasK2001** [comment, marked as answer]
Dear Salma,

looks like some Version incompatibility. Maybe you opened the original tutorial files it in Version 10? If yes, than I think it could help to rebuild the building block in Version 10. There have been some major releases in-between the tutorial files where build and the actual PK-sim version.

Hope this helps

Best, Tobias

**Reply by bahsa505** [reply]
Dear Tobias

This is exactly the case :) It works if I create the blocks myself. 

Thanks a lot
Salma

**Comment by Slow-Walker66** [comment]
Hi, 
I am working on this tutorial too.
However, I have some troubles to understanding some ODE formula in the text, could you kindly explain it to me?
![image](https://user-images.githubusercontent.com/87760070/222713219-024b6249-8902-45de-9fb5-dade867d50fc.png)
For example, why it was '-log(2)' ?

Thanks a lot!
